util/docker/inner/watch.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
```

Comment:
I'm a little confused about the directory structure - `util/docker/inner` makes me think that a container is running inside another container. What about:

- `util/docker-dev/` for the server
- `util/docker-jenkins/` for the test runner